### ISSUE TITLE
Update for VCV 2?

### ISSUE THREAD

**Issue by trevormeier** [aptrn/stocaudio-modules#13]
Hi there—any plans to update Stocaudio modules for VCV Rack version 2?

**Comment by RichieHindle** [comment]
+1 from me!

I was delighted with how well Polyspread worked in this demo: https://youtu.be/sTIqb63t7e8?t=114

I haven't tested them thoroughly, but it looks like a simple increment to the main version number is all that's needed:

```
index ce27dee..65122fa 100644
--- a/plugin.json
+++ b/plugin.json
@@ -1,7 +1,7 @@
 {
   "slug": "stocaudio",
   "name": "stocaudio",
-  "version": "1.0.1",
+  "version": "2.0.1",
   "license": "BSD-3-Clause",
   "author": "Alessandro Petrone",
   "authorEmail": "contact@stocaudio.com",
```

**Comment by aptrn** [comment]
Hey, sure update it's on the way.
I tested VCV2 with my old modules yesterday. They  seem to actually work without any change beside incrementing versioning number indeed. 
I was planning to drop four of those (manSeq, btnSeq, polySlew and Clock) though. This because when I first released those modules there were not many polyphonic oriented utilities. Now I think those functions are pretty backed up and we don't need those anymore. Polyturing needs many fixes to be ok by the way, I'll check those out first.
